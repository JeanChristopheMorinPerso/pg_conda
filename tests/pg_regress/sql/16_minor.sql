BEGIN;
CREATE EXTENSION pg_conda;

SELECT conda_minor('1.2');
SELECT conda_minor('1');
SELECT conda_minor('1.2a.3'::condaversion);
ROLLBACK;
