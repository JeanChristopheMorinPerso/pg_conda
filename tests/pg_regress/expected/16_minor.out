BEGIN;
CREATE EXTENSION pg_conda;
SELECT conda_minor('1.2');
 conda_minor 
-------------
           2
(1 row)

SELECT conda_minor('1');
 conda_minor 
-------------
            
(1 row)

SELECT conda_minor('1.2a.3'::condaversion);
 conda_minor 
-------------
            
(1 row)

ROLLBACK;
